heudiconv/info.py

```diff
@@ -11,7 +11,7 @@
 REQUIRES = [
     'nibabel',
     'pydicom',
-    'nipype>=0.12.0',
+    'nipype>=1.0.0',
     'pathlib',
     'dcmstack>=0.7',
```

Comment:
This should fix the test. `lxml` did not set a `python_requires` when they broke 3.4 compatibility.

```suggestion
    'dcmstack>=0.7',
    'lxml<4.4.0; python_version == "3.4"',
```

Reply:
Oh, looks like @mgxd got here first.

Reply:
> @yarikoptic - are you okay with dropping support for 0.x nipype? I am.

yes -- let's look forward!

re lxml -- should we just stop carrying about 3.4?  Even Debian stable has 3.7 nowadays ;)

Reply:
I'm all for removing 3.4 from our testing suite!!!